migrations/1680527289416-remove-systems-permission-of-admins.js

```diff
@@ -0,0 +1,94 @@
+const mongoose = require('mongoose');
+// eslint-disable-next-line no-unused-vars
+const { alert, error, info } = require('../src/logger');
+
+const { connect, close } = require('../src/utils/database');
+
+const Roles = mongoose.model(
+	'roles200725',

```

Comment:
This should be a unique name. I recommend using `collectionJJJJMMDDHHmm`. There is sadly no standard to it...